conan/tools/microsoft/visual.py

```diff
@@ -27,9 +27,11 @@ def _write_conanvcvars(conanfile, group):
         return
 
     compiler = conanfile.settings.get_safe("compiler")
-    cvars = None
+    vcvars = None
     if compiler == "intel":
-        cvars = intel_compilervars_command(conanfile)
+        vcvars = intel_compilervars_command(conanfile)
+    elif compiler == "intel-cc":
```

Comment:
I am not convinced this belongs here. The ``CMakeToolchain`` is already generating the intel vars script. Isn't a bit weird that you call ``VCVars().generate()`` and it will create a intel file instead of a vcvars one?

Reply:
Yes, you're right. Removing it from here. Indeed, it's already integrated into `MSBuildToolchain` as a possible new toolset.